src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Threading.Tasks;
+using ApiView;
+using APIViewWeb.Models;
+using APIViewWeb.Repositories;
+using APIViewWeb.Respositories;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Octokit;
+
+namespace APIViewWeb.Repositories
+{
+    public class PullRequestManager
+    {
+        static readonly HttpClient devopsClient = new();
+        static readonly GitHubClient githubClient = new(new Octokit.ProductHeaderValue("apiview"));
+        static readonly string ARTIFACT_GET_URL = "https://dev.azure.com/azure-sdk/public/_apis/build/builds/{buildId}/artifacts?artifactName={artifactName}&api-version=4.1";
```

Comment:
Should this be pulled into some configuration somewhere?

Reply:
As we start to think about enable this for other repos like the private specs repo we might need to support pulling from non-public builds.

Reply:
Done. Adding it into a config param with a helper method that can pull REST API url based on the repo name